Code/Framework/AzToolsFramework/AzToolsFramework/Editor/EditorPreferenceGetter.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+namespace AzToolsFramework
```

Comment:
A similar file was recently added to handle prefab editor preferences. You can merge these changes into that file to keep all editor prefab preferences in one place : https://github.com/o3de/o3de/blob/development/Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabEditorPreferences.h